great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -28,6 +28,7 @@ class GeCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
         "expectation_suite": "suite",
         "rendered_data_doc": "rendered_data_doc",
         "data_context_variables": "data_context_variables",
+        "profiler": "profiler_config",
```

Comment:
```suggestion
        "profiler": "profiler",
```

but happy to change this in cloud

Reply:
nope happy to modify on my end to make it consistent with Cloud! fixing now